edge-agent/docker/linux/arm32v7/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG base_tag=1.0.4-linux-arm32v7
+ARG base_tag=1.0.6-linux-arm32v7
```

Comment:
Just want to confirm that 1.0.5 is used in master and 1.0.10 branch when upgrade netcore to 3.1, right?

Reply:
That's correct.